### DISCUSSION TITLE
Delay between sysex messages

### DISCUSSION THREAD

**Discussion by Colpocorto** [RomanKubiak/ctrlr#176] [General]
Hi,

I'm working on my first CTRLR and noticed that there is a huge delay between each sysex message sent to the synth (about 500ms or so).

I haven't found any parameter or function to prevent this to happen. What am I doing wrong?
Thank you in advance.

**Comment by Ted-juh** [comment]
Did you try the setting in the “general” preference pane called: Global delay for midi message?

**Reply by Colpocorto** [reply]
Yes, I did but it didn't work.

Anyway, I've managed to fix it by modifying that value directly on the XML file.
After changing the XML and reloading the panel, the parameter seems to work again as intended. 

Thank you for your help.
